aiida/orm/node/process/calculation/calcjob.py

```diff
@@ -2161,8 +2156,8 @@ def submit_test(self, folder=None, subfolder_name=None):
         else:
             subfolder_basename = subfolder_name
 
-        # Find a new subfolder.
-        # I do not user tempfile.mkdtemp, because it puts random characters
+        ## Find a new subfolder.
+        ## I do not user tempfile.mkdtemp, because it puts random characters
```

Comment:
Why two # characters? Either also for the lines below or don't add the second #

Reply:
Mistake. In first commit of this branch, I copied over content of `JobCalculation` and already cleaned it a bit such as these double comment characters. But then in following commits I changed `JobCalculation` a bit and to make sure that I included everything in this PR, I simply pasted the `JobCalculation` code once more in `CalcJobNode`, which undid my removal of the two characters. Long story short, I will remove them again :)